spec/features/admin_can_view_items_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+RSpec.describe 'Admin', type: :feature do
+  create_category
+  create_brand
+  create_items
+  create_user
+  create_rental
+  create_admin
+  feature 'admin' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.